src/main/java/com/google/ehub/data/EntertainmentItemList.java

```diff
@@ -0,0 +1,27 @@
+package com.google.ehub.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class EntertainmentItemList {
+  private final List<EntertainmentItem> itemList;
+  private final String pageCursor;
+
+  public EntertainmentItemList() {
+    itemList = new ArrayList<>();
+    pageCursor = "";
+  }
```

Comment:
do you need this constructor?

Reply:
Yeah I didn't need to; forgot that Java doesn't call constructor if the object is left without being initialized.